### DISCUSSION TITLE
Admin panel tracks usage data without consent

### DISCUSSION THREAD

**Discussion by Paratron** [strapi/strapi#6944] [Legacy Topics]
I took a look into the network panel of the strapi admin and noticed that it issues a lot of requests to `https://analytics.strapi.io/track` and sends usage information without asking for consent beforehand.

This is an _absolute_ no go. There is ZERO information about that strapi does this and there is no mention in the docs about how to disable the tracking.

Not only is this a shady move against your users but this is even illegal according to european data protection laws and could bring you into legal trouble. Heck, it could even bring the companies into trouble that are using strapi.

**Comment by derrickmehaffy** [comment]
https://strapi.io/documentation/v3.x/global-strapi/usage-information.html#collected-usage-information

**Comment by Paratron** [comment]
Small update, I just found a page buried deep down into the strapi documentation that mentiones the tracking.

https://strapi.io/documentation/v3.x/global-strapi/usage-information.html

This is still a problem since its enabled by default without any heads up to the user.

**Reply by Gusterson** [reply]
> This is still a problem since its enabled by default without any heads up to the user.

Thanks for bringing this to my attention. Disabled.

**Comment by Paratron** [comment]
Oh, you already closed this without any comment. Well thank you -.-

**Comment by Aurelsicoko** [comment]
I would like to give you a complete answer. As other open-source software (Node, Gatsby, Next, etc),  we want to improve our product based on the usage. To do that, we need to collect basic data. Any open-source software is provided "as is" (see the MIT license  https://en.wikipedia.org/wiki/MIT_License). We do not collect any personal data, everything is anonymously using a random generated unique ID. We are 100% compliant with GDPR and worked with open-source lawyers experts to follow best-in-class practices.

The documentation clearly explains what we collect, the reasons why we collect them, and how to opt-out in no time.
